Exclude narrower terms link fix, refs #13544

mInnes-archives opened the pull request:
PR addresses a routing bug with the "Exclude narrower terms" button in the "Related authority records" tab in term search. It currently routes back to the "Related Archival descriptions tab" rather than staying in the correct tab.

Including the correct action when constructing the url fixes the issue.

147279b05dfcfa46735db5c4e305bff6f97587fa -> I was getting a message about the config file name being incorrect so I renamed it, but I see the issue is more complicated after seeing #1331. My changes aren't triggering the fixer anyway.

mInnes-archives commented:
Just realising this now, but staying on the correct tab seems to be an issue for a number of different link elements. I'll switch this to a WIP so to track them all down.

mInnes-archives commented:
The links in the treeview menu still reset the tabs, but that's not an easy fix as far as I can tell and I'm not sure if I'd even call this behaviour an 'issue'. Everything else is fine.

jraddaoui reviewed (approved):
Nice find @mInnes-archives, thank you!

Looks great, I've created https://projects.artefactual.com/issues/13544 for the reference.

mInnes-archives committed:
Exclude narrower terms link fix, refs #13544

Squashed commit of the following:

commit d953bf915a42544554385514a6ea1d6f2ae2e678
Author: Matt Innes <matthew.f.innes@gmail.com>
Date:   Thu Jul 15 17:28:27 2021 -0400

    Simplify logic

commit e9e5b1b4c2718d7c45eb5dc0ce1b75ad3263c6eb
Author: Matt Innes <matthew.f.innes@gmail.com>
Date:   Thu Jul 15 16:44:15 2021 -0400

    Fix filter tag link

commit a93630212deb3880ac08233c17bafdc329a42aa7
Author: Matt Innes <matthew.f.innes@gmail.com>
Date:   Thu Jul 15 16:14:08 2021 -0400

    Revert "Run php-cs-fixer"

    This reverts commit 147279b05dfcfa46735db5c4e305bff6f97587fa.

commit 26a0bca29d07066e228372024e7181d8616e0ca7
Author: Matt Innes <matthew.f.innes@gmail.com>
Date:   Thu Jul 15 16:06:24 2021 -0400

    Run php-cs-fixer

commit 31649529369d595f6d8183c8c2eb2c941a2c7471
Author: Matt Innes <matthew.f.innes@gmail.com>
Date:   Thu Jul 15 16:04:15 2021 -0400

    Fix exclude narrower terms link

jraddaoui commented:
Thanks @mInnes-archives!
